sql/09-mutual-likes.sql

```diff
@@ -0,0 +1,81 @@
+SELECT DISTINCT
+	student1.name,
+	student1.grade,
+	student2.name,
+	student2.grade
+FROM
+	student
+AS
+	student1
+JOIN
+	student_like
+ON
+	student1.id = student_like.liker_id
+JOIN
+	student
+AS
+	student2
+ON
+	student_like.likee_id = student2.id
+WHERE 
+	student1.id IN (
+		SELECT likee_id FROM student_like
+		)
+ORDER BY
+	student1.name;
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
This much whitespace doesn't make it any clearer why you're breaking up the queries, so it isn't really helpful  - A better way to separate them would be with a comment line to explain the difference, or by making 09-mutual-likes a directory and separating the queries into different sql files inside that directory.